### ISSUE TITLE
training issue!

### ISSUE THREAD

**Issue by Chadkowski** [282857341/nnFormer#102]
Hi. everytime I use nnFormer to train, the whole learning process starts behaving very weird after 3-4 epochs. Please see below:
(this happens regardless of dataset but this one was tested on synapse dataset)

2022-11-22 23:23:13.618784: 
 
2022-11-22 23:23:13.619194: 
epoch:  0 
2022-11-22 23:24:55.847692: train loss : 0.4745 
2022-11-22 23:25:04.270859: validation loss: 0.3916 
2022-11-22 23:25:04.271266: Average global foreground Dice: [0.0, 0.0001320296010365524, 2.274101208874301e-05, 0.0, 0.5969908676653919, 0.0, 0.0004676893599727684, 0.0] 
2022-11-22 23:25:04.271303: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-11-22 23:25:04.607909: lr: 0.009991 
2022-11-22 23:25:04.608021: current best_val_eval_criterion_MA is 0.07470 
2022-11-22 23:25:04.608056: current val_eval_criterion_MA is 0.0747 
2022-11-22 23:25:04.608111: This epoch took 110.988639 s
 
2022-11-22 23:25:04.608142: 
epoch:  1 
2022-11-22 23:26:39.660466: train loss : 0.4107 
2022-11-22 23:26:48.049546: validation loss: 0.3228 
2022-11-22 23:26:48.049927: Average global foreground Dice: [5.155274283790632e-06, 0.10146689407960259, 0.07402830708351602, 0.0, 0.5865517051479923, 0.0, 0.190082921281602, 0.0] 
2022-11-22 23:26:48.049962: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-11-22 23:26:48.361166: lr: 0.009982 
2022-11-22 23:26:48.361290: current best_val_eval_criterion_MA is 0.07470 
2022-11-22 23:26:48.361320: current val_eval_criterion_MA is 0.0791 
2022-11-22 23:26:48.541745: saving checkpoint... 
2022-11-22 23:26:49.902999: done, saving took 1.54 seconds 
2022-11-22 23:26:49.913620: This epoch took 105.305442 s
 
2022-11-22 23:26:49.913674: 
epoch:  2 
2022-11-22 23:28:23.650122: train loss : nan 
2022-11-22 23:28:32.050157: validation loss: 0.2625 
2022-11-22 23:28:32.050549: Average global foreground Dice: [8.664553959592852e-07, 0.035751748251748254, 5.627967535285733e-05, 0.0, 0.7476544526585733, 9.193226920710256e-06, 0.20471679386411343, 0.0] 
2022-11-22 23:28:32.050584: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-11-22 23:28:32.350800: lr: 0.009973 
2022-11-22 23:28:32.350904: current best_val_eval_criterion_MA is 0.07910 
2022-11-22 23:28:32.350929: current val_eval_criterion_MA is 0.0836 
2022-11-22 23:28:32.484009: saving checkpoint... 
2022-11-22 23:28:33.946634: done, saving took 1.60 seconds 
2022-11-22 23:28:33.959197: This epoch took 104.045484 s
 
2022-11-22 23:28:33.959269: 
epoch:  3 
2022-11-22 23:30:09.360566: train loss : nan 
2022-11-22 23:30:17.759997: validation loss: 0.3103 
2022-11-22 23:30:17.760396: Average global foreground Dice: [0.18516268021874827, 0.05451979106578396, 0.00011326764572626299, 0.0, 0.6808126343402319, 0.00023126412611603676, 0.34789594875600277, 0.0013141655534632897] 
2022-11-22 23:30:17.760430: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-11-22 23:30:18.065005: lr: 0.009964 
2022-11-22 23:30:18.065115: current best_val_eval_criterion_MA is 0.08360 
2022-11-22 23:30:18.065141: current val_eval_criterion_MA is 0.0911 
2022-11-22 23:30:18.211870: saving checkpoint... 
2022-11-22 23:30:19.666188: done, saving took 1.60 seconds 
2022-11-22 23:30:19.686091: This epoch took 105.726789 s
 
2022-11-22 23:30:19.686258: 
epoch:  4 
2022-11-22 23:31:57.169777: train loss : nan 
2022-11-22 23:32:05.593204: validation loss: 1.4975 
2022-11-22 23:32:05.593626: Average global foreground Dice: [0.19781398936287933, 0.0, 0.03799706011142289, 0.0, 0.036336266863399164, 0.023039737116651585, 5.222491180518019e-06, 0.0] 
2022-11-22 23:32:05.593674: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-11-22 23:32:05.909654: lr: 0.009955 
2022-11-22 23:32:05.909767: current best_val_eval_criterion_MA is 0.09110 
2022-11-22 23:32:05.909795: current val_eval_criterion_MA is 0.0857 
2022-11-22 23:32:05.909823: This epoch took 106.223495 s
 
2022-11-22 23:32:05.909852: 
epoch:  5 
2022-11-22 23:33:41.345607: train loss : nan 
2022-11-22 23:33:49.781050: validation loss: 1.8927 
2022-11-22 23:33:49.781520: Average global foreground Dice: [0.1855083287413658, 0.0, 0.01310647396050735, 0.0, 0.04130637519675208, 0.032110824761916984, 0.0, 0.0] 
2022-11-22 23:33:49.781560: (interpret this as an estimate for the Dice of the different classes. This is not exact.) 
2022-11-22 23:33:50.090640: lr: 0.009946 
2022-11-22 23:33:50.090751: current best_val_eval_criterion_MA is 0.09110 
2022-11-22 23:33:50.090780: current val_eval_criterion_MA is 0.0805 
2022-11-22 23:33:50.090809: This epoch took 104.180931 s

**Comment by Chadkowski** [comment]
running on CUDA 11.8 and torch 1.13.0 with driver 520.56.06 on ubuntu
